src/test/java/uk/gov/hmcts/reform/finrem/notifications/controllers/NotificationControllerTest.java

```diff
@@ -20,11 +20,15 @@
 import uk.gov.hmcts.reform.finrem.notifications.service.EmailService;
 
 import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
```

Comment:
Couple unused imports here, be worth removing these for cleaner code

Reply:
Removed